src/awkward/_v2/contents/content.py

```diff
@@ -308,11 +329,11 @@ def format_slice(x):
             raise IndexError(
                 """cannot slice
 
-    {0}
+{0}
 
 with
 
-    {1}
+{1}
```

Comment:
I wanted the two "semi-quoted" things (what you're slicing and what you're slicing it with) to be indented, to be separated from the rest of the text. The string representation of an `ak.Array` will never be multi-line, and it's hard to imagine a situation in which a slice would be. They should look okay indented.